viral/multiple_sessions.py

```
@@ -43,6 +49,7 @@
 )
 
 import seaborn as sns
+import pandas as pd
```

Ruff (0.13.2)

Redefinition of unused `pd` from line 13

Remove definition: `pd`

(F811)